locales/en.yaml

```diff
@@ -2,103 +2,100 @@ system:
   commands:
     groups:
       basic:
-        name: 基础命令
+        name: Basic Commands
         commands:
           start:
-            help: Start interacting with the bot
+            help: Begin interacting with the bot
           help:
-            help: Get help
+            help: Obtain assistance
             message: |
-              Hi! 👋 Welcome using Insights Bot!
+              Greetings! 👋 Welcome to using Insights Bot!
 
               I currently support these commands:
 
               {{ .Commands }}
           cancel:
-            help: Cancel the current activated operation
-            alreadyCancelledAll: There is no activated operation to cancel
+            help: Terminate the current operation
+            alreadyCancelledAll: No ongoing operations to terminate
 
 modules:
   telegram:
     chatMigration:
-      notification: |
-        {{.Name}} @{{.Username}} 监测到您的群组已从 <b>群组（group）</b> 升级为了 <b>超级群组（supergroup）</b>，届时，群组的 ID 将会发生变更，<b>现已自动将过去的历史记录和数据留存自动迁移到了新的群组 ID 名下</b>，之前的设置将会保留并继续沿用，不过需要注意的是，由于 Telegram 官方的限制，迁移事件前的消息 ID 将无法与今后发送的消息 ID 相兼容，所以当下一次总结消息时将不会包含在迁移事件发生前所发送的消息，由此带来的不便敬请谅解。
+  notification: |
+    {{.Name}} @{{.Username}} has observed your group's transition to a <b>supergroup</b>, leading to a change in the group ID. Rest assured, we've smoothly transitioned all historical data to the new ID, maintaining all your settings unaltered. Due to Telegram's limitations, message IDs from before the upgrade won't match those sent after, and will thus be excluded from future summaries. We regret any inconvenience caused by this adjustment.
 
-    welcome:
-      messageSuperGroup: |
-        🤗 欢迎使用 @{{.Username}}！
+welcome:
+  messageSuperGroup: |
+    🤗 Welcome to @{{.Username}}!
 
-        - 如果要让我帮忙阅读网页文章，请直接使用开箱即用的命令 /smr@{{.Username}} <code>要阅读的链接</code>；
+    - Use /smr@{{.Username}} <code>article link</code> for web article summaries.
 
-        - 如果想要我帮忙总结本群组的聊天记录，请以<b>管理员</b>身份将我配置为本群组的管理员（可以关闭所有权限），然后在<b>非匿名和非其他身份的身份</b>下（推荐，否则容易出现权限识别错误的情况）发送 /configure_recap@{{.Username}} 来开始配置本群组的聊天回顾功能。
+    - For chat history summaries, appoint me as admin (with or without permissions) using a <b>non-anonymous identity</b> to bypass permission detection issues. Then, start /configure_recap@{{.Username}} to configure chat recap.
```

Comment:
```suggestion
    - For chat history summaries, please assign me as admin (all permissions can be omitted) using a <b>non-anonymous identity</b> (recommended, otherwise permission validation may fail. Then, start /configure_recap@{{.Username}} to configure chat recap.
```